app/views/state_file/questions/esign_declaration/edit.html.erb

```diff
@@ -0,0 +1,17 @@
+<% state_name = States.name_for_key(params[:us_state].upcase) %>
+<% title = t(".title", state_name: state_name) %>
+<% content_for :page_title, title %>
+
+<% content_for :card do %>
+  <h1 class="h2"><%= title %></h1>
+  <%= params[:us_state] == "ny" ? t(".ny.body_html") : t(".az.body_html") %>
```

Comment:
Just an idea/observation: I feel like we could use a helper method for checking the state param.